src/foam/comics/v2/DAOControllerConfig.js

```diff
@@ -138,8 +138,11 @@ foam.CLASS({
     },
     {
       class: 'FObjectProperty',
-      of: 'foam.comics.v2.CRUDActionsAuth',
-      name: 'CRUDActionsAuth'
+      of: 'foam.comics.v2.CRUDActionsPredicate',
```

Comment:
What advantage do we get by making this be a nested Object rather than just adding three Predicate properties here? Are we using the CRUDActionsPredicate somewhere else?

Reply:
Fixed

Reply:
Why do we have this permission and still have 'created/edit/deletedEnabled' above?